.github/workflows/update-client.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         git config --global user.name "Hypothesis GitHub Actions"
         git config --global user.email "hypothesis@users.noreply.github.com"
         tools/update-client
-        echo "::set-output name=ref::$(git rev-parse HEAD)"
+        echo "ref=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
```

Comment:
See https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/